### PR TITLE
Nfdiv 721

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-721

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/divorce/common/model/Application.java [hunk 1]
```diff
@@ -270,6 +270,18 @@
     @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS")
     private LocalDateTime dateSubmitted;
 
+    @CCD(
+        label = "Is the information provided on the case correct?",
+        access = {DefaultAccess.class}
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:275]: shouldn't the access be set to applicant2?

File: src/main/java/uk/gov/hmcts/divorce/common/model/Application.java [hunk 2]
```diff
@@ -270,6 +270,18 @@
...
+    )
+    private YesOrNo applicant2ConfirmApplicant1Information;
+
+    @CCD(
+        label = "Explain what is incorrect or needs changing.",
+        access = {DefaultAccess.class}
+    )
+    private String applicant2ExplainsApplicant1IncorrectInformation;
```
Comment (linusnorton) [inline src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:283]: isn't this field shared between app1 and app2?

Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:283]: in what way?
I thought only applicant 2 confirms and applicant 1 is the one doing most of the details.

File: src/main/java/uk/gov/hmcts/divorce/common/model/Application.java (hunk 1 above)
Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:275]: im not sure how thats done tbh, it seems like all of the fields are set to DefaultAccess.class

File: src/main/java/uk/gov/hmcts/divorce/common/model/Application.java (hunk 2 above)
Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:283]: I believe it can go both ways. Lets check with Michael

Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:283]: explained how Michael said its different for applicant 1 reviewing it.

File: src/main/java/uk/gov/hmcts/divorce/common/model/Application.java (hunk 1 above)
Comment (velimd) [reply src/main/java/uk/gov/hmcts/divorce/common/model/Application.java:275]: discussed, ticket for this.

**Review by linusnorton — APPROVED** [review]

**Pull request merged**